vpr/src/timing/slack_evaluation.cpp

```diff
@@ -49,11 +49,11 @@ SetupSlackCrit::SetupSlackCrit(const AtomNetlist& netlist, const AtomLookup& net
 }
 
 SetupSlackCrit::~SetupSlackCrit() {
-    VTR_LOG("Incr Slack updates %zu in %g sec\n", incr_slack_updates_, incr_slack_update_time_sec_);
```

Comment:
This doesn't belong in this PR?

Reply:
You're right but I think it does belong in a PR. It seems to be debug code that made it into master, and clutters the router output. I can remove it from this, thanks!

Reply:
I will put this in before we merge, but just for testing purposes I'm leaving this out.